packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/TransactionsCompletedRow.tsx

```diff
@@ -34,6 +34,10 @@ const TransactionsCompletedRow: React.FC<TProps> = ({ accounts, transaction, wal
 
     const displayCurrency = wallet?.currency_config?.display_code || 'USD';
     const displayWalletName = `${displayCurrency} Wallet`;
+    const displayActionType =
+        wallet.is_virtual && ['deposit', 'withdrawal'].includes(transaction.action_type)
+            ? 'Reset balance'
+            : transaction.action_type.replace(/^\w/, c => c.toUpperCase());
```

Comment:
will this work with localisation, or will that need refactoring to work with localisation (I know this mapping/replacing was introduced before this PR)

Reply:
Good point, it's not a literal; I hope to redo this code in a PR for localising Transactions